### ISSUE TITLE
Double Tap Button/Link Bug iOS Devices

### ISSUE THREAD

**Issue by TomAllenBerg** [surveyjs/survey-library#2102]
### Are you requesting a feature, reporting a bug or asking a question?

reporting a bug

### What is the current behavior?

Upon using a survey.js survey in the modern theme on an ios device with safari you must double click the next, previous and complete buttons in order for them to work. Here is an article that may help with fixing this bug: https://css-tricks.com/annoying-mobile-double-tap-link-issue/

Edit: It appears this may more likely be the issue: https://mobile.twitter.com/simevidas/status/779161023790116865

Edit2: Perhaps you could add one of these implementations into a new survey js version: https://medium.com/turo-engineering/react-stick-to-bottom-accessible-safari-fixed-bottom-elements-9ed761fbeecb

### What is the expected behavior?

The expected behavior is to only have to click a button/link once in order for it to work.

### Specify your

* browser: Safari
* surveyjs platform (angular or react or jquery or knockout or vue): jquery

**Comment by tsv2013** [comment]
I've just check it in iPhone 8 under iOS 13.3.1 with this https://surveyjs.io/Examples/Library?id=questiontype-image&platform=jQuery&theme=modern and this https://surveyjs.io/Examples/Library?id=real-nps&platform=jQuery&theme=modern online examples from our site. 

The "Complete" button works ok for me - from the first click. Do I miss something? Is the issue reproduced on specific device or specific browser version?

**Comment by dmitrykurmanov** [comment]
also I can't manage to reproduced the problem on MacBook and IPad

**Comment by TomAllenBerg** [comment]
Apologies for the delayed response. I have encountered this issue specifically with an old iPhone 5 running iOS 10.3.4. I am unsure if this behavior has changed in later iOS versions. The issue is that with a survey I had created the 'Next' and 'Previous' buttons were within the bottom 44 pixels of the screen. Upon screen tap in this area of the screen the 'menu bar' pops up and the button is not clicked on. Here is a gif I found that illustrates this: 
![RlJdE](https://user-images.githubusercontent.com/54914160/80783194-eab62a80-8b46-11ea-90c1-9a246efedc8e.gif)

Anyways, I no longer believe there is a solution that survey js should be responsible for fixing. Instead, it is up to the individual to work around this issue. I apologize for any lost time; however, someone else may come across this in the future and find it beneficial.
